src/ztex/ztex.h

```diff
@@ -42,6 +43,7 @@
 #define CAPABILITY_MAC_EEPROM 0,6
 // Capability index for multi FPGA support.
 #define CAPABILITY_MULTI_FPGA 0,7
+// Copy-pasted from ZTEX SDK! (java)
```

Comment:
It's unclear how much is copy-pasted. If it's just one line, then it's confusing why the comment was needed. If it's much more than that, there can be copyright+licensing concerns.

Reply:
I've copied definitions of bits in configuration bytes from ZTEX specific USB descriptor. There was some issue, I've commented with exclamation to return later. I'll take a look and write more descriptive comment.